restart.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="ENGLISH">
+<head>
+    <meta charset="UTF-8">
+    <title> Becks Webpage</title>
+    <link rel="stylesheet" href="restart.css" type="text/css"/>
+</head>
+<style> 
+h1,h2,h3 {text-align: center;}
+p {text-align: left;}
+</style>
+<body>
+
+<div id="main">
+<h1 id="title"> Becks:
+</h1>
+
+ <p id="def"> 
+    /bɛks/ <i>noun</i> NORTHERN ENGLISH </p>
+     <p id="def"><em>
+        A friendly, curious, passionate (sometimes feisty) , strong, hard-working woman
+     </em></p>
+     <img src="images/IMG_2736.jpg" alt="Becks">
+     
+</div>
+<!--<div>
+<p></p>I love reading
+I love music
+I aspire to be a good mum
+I am a good listener
+I like using my brain -->
+
+
+<div id="music">x
+<h2> Music & Me</h2>
+<p id=def>Music is food for my soul.It turns off my inner chatter and recalibrates me </p>
```

Comment:
Attributes values should be wrapped in speech marks. 
Naming convention. Naming ID/Classes to explain exactly what the element is makes life easy for the developer as the project continues to grow into multiple pages. 
```suggestion
<p id="becks-definition">Music is food for my soul.It turns off my inner chatter and recalibrates me </p>
```